sdk/core/src/main/java/com/klaviyo/core/KlaviyoException.kt

```diff
@@ -18,16 +22,20 @@ abstract class KlaviyoException(final override val message: String) : Exception(
  * Take care not to nest [safeCall] invocations, because the inner exception
  * will not halt execution of the outer method.
  */
-inline fun <T> safeCall(block: () -> T): T? {
-    return try {
-        block()
-    } catch (e: KlaviyoException) {
-        // KlaviyoException is self-logging
-        null
-    }
+fun <ReturnType> safeCall(
+    errorQueue: Queue<Operation<ReturnType>>? = null,
+    block: Operation<ReturnType>
+): ReturnType? = try {
+    block()
+} catch (e: KlaviyoException) {
+    // KlaviyoException is self-logging
```

Comment:
What does self logging mean?

Reply:
`KlaviyoException` logs an error message from its initializer... considering this is like the 3rd time someone's asked me that I feel like I should change that though if its causing (minor) confusion